test.js

```diff
@@ -1,20 +1,25 @@
 import test from 'ava';
 import {execa} from 'execa';
 
+// Tests only checks that opening doesn't return an error, not that the correct page was opened.
+// These have to be manually verified.
```

Comment:
Is it worth adding something like `puppeteer` to test that the correct pages are opened?

Reply:
Nah, not worth the effort. It's easy enough to quickly manually check.